docs/make.jl

```diff
@@ -1,7 +1,14 @@
-using Documenter, DocumenterTools
+using Documenter, DocumenterTools, DocumenterLaTeX, DocumenterMarkdown
 
 makedocs(
     modules = [Documenter, DocumenterTools],
+    # format = Documenter.HTML(
+    #     # Use clean URLs, unless built as a "local" build
+    #     prettyurls = !("local" in ARGS),
+    #     canonical = "https://juliadocs.github.io/Documenter.jl/stable/",
+    # ),
+    format = Documenter.Writers.LaTeXWriter.LaTeX(engine = "docker"),
```

Comment:
Should we maybe set up a second Documentation build stage which deploys the PDFs to `gh-pages-pdf` (assuming updates to `deplydocs` are on the agenda)?